wordfreq_builder/wordfreq_builder/word_counts.py

```diff
@@ -32,36 +32,54 @@ def count_tokens(filename):
     return counts
 
 
-def read_freqs(filename, cutoff=0, lang=None):
+def read_values(filename, cutoff=0, lang=None):
     """
-    Read words and their frequencies from a CSV file.
+    Read words and their frequency or count values from a CSV file. Returns
+    a dictionary of values and the total of all values.
 
-    Only words with a frequency greater than or equal to `cutoff` are returned.
+    Only words with a value greater than or equal to `cutoff` are returned.
 
-    If `cutoff` is greater than 0, the csv file must be sorted by frequency
+    If `cutoff` is greater than 0, the csv file must be sorted by value
     in descending order.
 
-    If lang is given, read_freqs will apply language specific preprocessing
+    If lang is given, it will apply language specific preprocessing
     operations.
     """
-    raw_counts = defaultdict(float)
+    values = defaultdict(float)
     total = 0.
     with open(filename, encoding='utf-8', newline='') as infile:
         for key, strval in csv.reader(infile):
             val = float(strval)
+            key = fix_text(key)
             if val < cutoff:
                 break
             tokens = tokenize(key, lang) if lang is not None else simple_tokenize(key)
             for token in tokens:
                 # Use += so that, if we give the reader concatenated files with
                 # duplicates, it does the right thing
-                raw_counts[fix_text(token)] += val
+                values[token] += val
                 total += val
+    return values, total
 
-    for word in raw_counts:
-        raw_counts[word] /= total
 
-    return raw_counts
+def read_freqs(filename, cutoff=0, lang=None):
```

Comment:
They differ more in the next PR: there is a fix to English apostrophes that should only be applied to frequencies.